bezier_library/include/bezier_library/bezier_library.hpp

```diff
@@ -76,6 +76,8 @@ class Bezier
   /** @brief Initialized constructor
    *  @param[in] filename_inputMesh filename of input poly data (input mesh)
    *  @param[in] filename_defectMesh filename of defect poly data (defect mesh)
+   *  @param[in] lean_angle_axis axis to provide an angle for effector
+   *  @param[in] angle_value value to give to axis provides below
```

Comment:
@param[in] lean_angle_axis rotation axis for the lean angle
@param[in] angle_value rotation value (in radians) around lean_angle_axis